build.sbt

```diff
@@ -97,11 +80,7 @@ lazy val reflect = project
   .in(file("scalafix-reflect"))
   .settings(
     moduleName := "scalafix-reflect",
-    isFullCrossVersion,
-    libraryDependencies ++= Seq(
-      "org.scala-lang" % "scala-compiler" % scalaVersion.value,
-      "org.scala-lang" % "scala-reflect" % scalaVersion.value
```

Comment:
Those two also are removed. Not sure if it's better to hide the fact that we depend on the compiler.

Reply:
In that case, since scalafix-reflect is full cross-version we wouldn't want to end up with wrong patch version brought transitively, it would be interesting to keep them. although https://github.com/scalacenter/scalafix/blob/48b201b89fa11edb7496dd923e188cad1b490184/build.sbt#L89 effectively ensure that the patch versions are aligned...